Samples/HelloWorld/OrleansHost/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace OrleansHost
+{
+    /// <summary>
+    /// Orleans test host
+    /// </summary>
+    public class Program
+    {
+        private static OrleansHostWrapper hostWrapper;
+        static int Main(string[] args)
+        {
+            int returnCode = StartSilo(args);
+
+            Console.WriteLine("Press Enter to terminate...");
```

Comment:
if the silo didn't start, you should terminate without waiting for input or calling shutdown